### PR TITLE
add pending state for canceling orders

### LINKED ISSUE #5557: Wire up `processing` to open orders when cancelling
cancelling an order can take time. need to show `processing` on the open order while it's getting cancelled.

### PR DESCRIPTION
**Pull request by andrewdamelio**
closes #5557

### CODE REVIEW

File: packages/augur-ui/src/modules/events/actions/add-update-transaction.ts [hunk 1]
```diff
@@ -243,15 +243,15 @@ export const addUpdateTransaction = (txStatus: Events.TXStatus) => async (
         break;
       }
       case CANCELORDER: {
-        const orderId = transaction.params[TX_ORDER_ID];
+        const orderId = transaction.params && transaction.params.order['orderId'];
```
Comment (bthaile) [inline packages/augur-ui/src/modules/events/actions/add-update-transaction.ts]: change TX_ORDER_ID to be `orderId` and use the constant.  doesn't look like it's used anywhere else

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🌠
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/l74hkzn**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/events/actions/add-update-transaction.ts (hunk 1 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/events/actions/add-update-transaction.ts]: updated

**Review by bthaile — APPROVED** [review]
Lgtm

**Pull request merged**